CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.1.4
+
+**Release date:** 2021-04-28
+
```

Comment:
This prerelease includes flux2 [v0.13.1](https://github.com/fluxcd/flux2/releases/tag/v0.13.1).